tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1398,20 +1405,21 @@ static uint8_t set_extended_advertising(const void *cmd, uint16_t cmd_len,
 }
 
 #if defined(CONFIG_BT_PER_ADV)
-static struct bt_data padv[10];
-static struct bt_le_per_adv_sync *pa_sync;
+static struct bt_data *per_adv_local[CONFIG_BT_EXT_ADV_MAX_ADV_SET];
+static uint8_t per_adv_local_len[CONFIG_BT_EXT_ADV_MAX_ADV_SET];
+static struct bt_le_per_adv_sync *pa_local_sync[CONFIG_BT_EXT_ADV_MAX_ADV_SET];
```

Comment:
Similar to broadcast, the number of sync we have is not related to `CONFIG_BT_EXT_ADV_MAX_ADV_SET`